packages/react/menu/src/Menu.tsx

```diff
@@ -301,23 +300,9 @@ const MenuRootContentNonModal = React.forwardRef((props, forwardedRef) => {
       trapFocus={false}
       disableOutsidePointerEvents={false}
       disableOutsideScroll={false}
-      onCloseAutoFocus={(event) => {
-        if (isPointerDownOutsideRef.current) {
-          event.preventDefault();
-        } else {
-          props.onCloseAutoFocus?.(event);
-        }
-      }}
-      onEscapeKeyDown={composeEventHandlers(props.onEscapeKeyDown, () => {
-        isPointerDownOutsideRef.current = false;
-      })}
-      onPointerDownOutside={composeEventHandlers(
-        props.onPointerDownOutside,
-        (event) => {
-          const originalEvent = event.detail.originalEvent as MouseEvent;
-          const isLeftClick = originalEvent.button === 0 && originalEvent.ctrlKey === false;
-          isPointerDownOutsideRef.current = isLeftClick;
-        },
+      onCloseAutoFocus={composeEventHandlers(
+        props.onCloseAutoFocus,
+        (event) => event.preventDefault(),
```

Comment:
`Menu` will now always expect auto focus will be managed manually (where before it was leaning into the implicit auto focus behaviour hence it working fine with dropdown in the previous config)